library/src/G4LatticeLogical.cc

```diff
@@ -457,7 +464,7 @@ G4LatticeLogical::MapV_elToP(G4int ivalley, const G4ThreeVector& v_e) const {
   G4double bandV = (fMassTensor.xx()*tempvec().x()*tempvec().x() +
   fMassTensor.yy()*tempvec().y()*tempvec().y() +
   fMassTensor.zz()*tempvec().z()*tempvec().z());
-  G4double gamma = 1/sqrt(1-bandV/GetElectronMass()*c_squared);
+  G4double gamma = 1/sqrt(1-bandV/GetElectronMass()/c_squared);
```

Comment:
Better to use parentheses for the denominator (multiple divisions are more likely to drive roundoff issues compared to a multiplication and single division).